Microsoft.Toolkit.Uwp/IncrementalLoadingCollection/IncrementalLoadingCollection.cs

```diff
@@ -196,10 +196,10 @@ public IAsyncOperation<LoadMoreItemsResult> LoadMoreItemsAsync(uint count)
             => LoadMoreItemsAsync(count, new CancellationToken(false)).AsAsyncOperation();
 
         /// <summary>
-        /// Clears the collection and reloads data from the source
+        /// Clears the collection and resets the page index
+        /// which triggers an automatic reload of the first page
         /// </summary>
-        /// <returns>This method does not return a result</returns>
-        public async Task RefreshAsync()
+        public void Refresh()
```

Comment:
This is a breaking change. Please mark the existing method obsolete and add the new Refresh. RefreshAsync can simply call Refresh, but it will need to continue with the existing method signature and return a Task as it is now. We can then remove it in 3.0.